runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1144,12 +1144,32 @@ NNFW_STATUS nnfw_session::train_prepare(const nnfw_train_info *info)
     return NNFW_STATUS_INVALID_STATE;
   }
 
-  // Use default info if info is null, otherwise use given info
-  (void)info;
+  if (info == nullptr)
+  {
+    std::cerr << "Error during model prepare training: info is null" << std::endl;
+    return NNFW_STATUS_UNEXPECTED_NULL;
+  }
```

Comment:
Please update `nnfw_train_prepare`'s comment in `nnfw_experimental.h` if you want to return `NNFW_STATUS_UNEXPECTED_NULL` when `info` is `nullptr`.

https://github.com/Samsung/ONE/blob/f03dd297f7d235b722f7151b0e9ca248744d4f47/runtime/onert/api/include/nnfw_experimental.h#L196-L200

Reply:
I will update the `nnfw_experimental.h` file. If we support nullptr information, we need to keep the trainingInfo in session object. However, it is only necessary if it is training. Since we do not fully support the training function yet, it would be good to support it by putting it into the session when the train function implementation is fully included in the ONERT in the future.